addons/controllers/suite_test.go

```diff
@@ -130,6 +134,12 @@ var _ = BeforeSuite(func(done Done) {
 	err = cniv1alpha1.AddToScheme(scheme)
 	Expect(err).NotTo(HaveOccurred())
 
+	err = cpiv1alpha1.AddToScheme(scheme)
+	Expect(err).NotTo(HaveOccurred())
+
+	err = capvv1beta1.AddToScheme(scheme)
```

Comment:
Cluster-API resources are already added in line 96

Reply:
oh I am adding the scheme for cpi and capv here. not CAPI   =)